connector-drawer/src/main/java/com/redhat/cloud/notifications/connector/drawer/model/DrawerNotificationToConnector.java

```diff
@@ -0,0 +1,19 @@
+package com.redhat.cloud.notifications.connector.drawer.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Collection;
+
+public record DrawerNotificationToConnector(
```

Comment:
I prefer keeping the current name, because it's just an POJO to pack things to connector, an also becase DrawerNotification entity already exists on engine :)